packages/salesforcedx-vscode-soql/package.json

```diff
@@ -36,7 +36,7 @@
     "@salesforce/apex-tmlanguage": "1.6.0",
     "@salesforce/core": "^2.15.2",
     "@salesforce/salesforcedx-utils-vscode": "51.3.0",
-    "@salesforce/soql-builder-ui": "0.0.34",
+    "@salesforce/soql-builder-ui": "0.1.0",
```

Comment:
Please add a test to make sure this actually fixes the original issue

Reply:
I think we're gonna need to revive the system-tests.  Because we really need to verify rendered html.